runtime_scan/pkg/orchestrator/scanwatcher/watcher.go

```diff
@@ -538,9 +538,14 @@ func (w *Watcher) reconcileAborted(ctx context.Context, scan *models.Scan) error
 	scan.EndTime = utils.PointerTo(time.Now())
 	scan.State = utils.PointerTo(models.ScanStateFailed)
 
-	err = w.backend.PatchScan(ctx, scanID, scan)
+	scanPatch := &models.Scan{
+		State:     scan.State,
+		EndTime:   scan.EndTime,
+		TargetIDs: scan.TargetIDs,
```

Comment:
If there was no change in the `TargetIDs` why it is being patched?

Reply:
That is due to "nullable: true" bug I've got the ticket to fix it, I'll try to get that pushed up today.

Reply:
What @sambetts-cisco said.